src/frontend/ikoner/EksternLenke.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+
+interface IEksternLenke {
+    className?: string;
+    height?: number;
+    width?: number;
+}
+
+export const EksternLenke: React.FC<IEksternLenke> = ({ className, width = 16, height = 16 }) => {
+    //<!-- Created with Method Draw - http://github.com/duopixel/Method-Draw/ -->
+    return (
+        <svg
+            width={width}
```

Comment:
Needs UU stuff, like aria-label and so forth.

Reply:
Good catch!